tests/phpunit/elementor/modules/usage/test-module.php

```diff
@@ -93,74 +96,191 @@ public function test_recalc_usage() {
 		$this->factory()->documents->publish_and_get();
 
 		// Clear global usage.
-		update_option( Module::OPTION_NAME, [] );
+		update_option( Module::ELEMENTS_OPTION_NAME, [] );
 
 		// Act.
 		$this->module->recalc_usage();
 
+		// Assert.
+		$this->assertEquals( 2, $this->get_global_elements_usage_by_document( $document )['button']['count'] );
+	}
+
+	public function test_add_document_usage__ensure_settings_added_to_global() {
+		// Act.
+		$document = $this->factory()->documents->publish_and_get( [
+			'meta_input' => [
+				Document::PAGE_META_KEY => [
+					'background_background' => 'red'
+				],
+			],
+		] );
 
 		// Assert.
-		$this->assertEquals( 2, $this->get_global_usage_by_document( $document )['button']['count'] );
+		$this->assertTrue( !! $this->get_global_settings_usage_by_document( $document ) );
```

Comment:
Remove the test, it's covered by the next test

Reply:
Handled: https://github.com/elementor/elementor/pull/13408/commits/72d34e808c04d2485e7cbff293ec534a9678b4df